.drone.yml

```diff
@@ -138,25 +80,17 @@ steps:
   - chmod +x ./kubectl
   - mv ./kubectl /usr/local/bin/kubectl
   - kubectl rollout restart deployment -n github-bot github-bot --token=$TOKEN --server=$API_SERVER_ARRD --insecure-skip-tls-verify
-  - kubectl rollout restart deployment -n github-runner github-runner-authorizer --token=$TOKEN --server=$API_SERVER_ARRD --insecure-skip-tls-verify
   environment:
     TOKEN:
         from_secret: token
     API_SERVER_ARRD:
         from_secret: api_server_addr
   when:
     instance:
-    - drone-publish.longhorn.io
+    - drone-publish.rancher.io
     branch:
-    - master
+    - harvester
     event:
     - push
 depends_on:
-- longhorn-bot
-- longhorn-github-runner-authorizer
-
-
-volumes:
-- name: socket
-  host:
-    path: /var/run/docker.sock
```

Comment:
the volume is needed for docker build.

Reply:
The harvester-bot pipeline already contains this volume. No docker build in the deploy-harvester-bot-images pipeline